environments/ttl/enable.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+
+import datetime
+from cloudify import manager
+from cloudify import ctx
+from cloudify.state import ctx_parameters as inputs
+
+SCHEDULE_FORMAT = '%Y-%m-%d %H:%M'
```

Comment:
@jrzeszutek did we really check the format , as for the documentation the format should be `YYYYMMDDHHMM+HHMM or YYYYMMDDHHMM-HHMM` 
https://docs.cloudify.co/api/v3.1/#start-execution

Reply:
I don't think it really matters. I tried all validations, I even tried to schedule a workflow manually from the GUI, but it still doesn't appear in the executions list as a scheduled workflow. But workflow is scheduled and it's being triggered at a scheduled time, so maybe let's not change it